storage/innobase/lock/lock0lock.cc

```diff
@@ -5364,6 +5374,23 @@ static void lock_cancel_waiting_and_release(lock_t *lock)
   lock_wait_end(trx);
   trx->mutex_unlock();
 }
+#ifdef WITH_WSREP
+/* if trx is waiting for a lock, cancel the waiting
+*/
+void lock_sys_t::cancel_lock_wait_for_trx(trx_t *trx)
```

Comment:
The comment should be either removed or be a verbatim copy of what is in `lock0lock.h`.